frontend/src/App.test.js

```diff
@@ -1,16 +1,29 @@
 import App from './App';
 import MyNavbar from "./components/MyNavbar/MyNavbar";
-import { shallow } from "enzyme";
+import { shallow, mount } from "enzyme";
 import { hasBeenRendered } from "./global";
+import SamplePage from './components/SamplePage/SamplePage';
+import { MemoryRouter } from 'react-router';
 
 const wrapper = shallow(<App />);
 
 describe("Test App Component", () => {
   it("should render", () => {
-      hasBeenRendered(wrapper);
+    hasBeenRendered(wrapper);
   });
 
   it("should render navbar", () => {
-      hasBeenRendered(wrapper.find(MyNavbar));
+    hasBeenRendered(wrapper.find(MyNavbar));
   });
+
+  // issues with enzyme + React 17: https://github.com/enzymejs/enzyme/issues/2462
```

Comment:
We might have to change to react 16 if we decide to start using mount. Lets discuss this as a team.